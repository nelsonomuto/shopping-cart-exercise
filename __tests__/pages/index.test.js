import React from "react"
import renderer from "react-test-renderer"
import IndexPage from "../../src/pages/index"

describe("IndexPage", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<IndexPage />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
