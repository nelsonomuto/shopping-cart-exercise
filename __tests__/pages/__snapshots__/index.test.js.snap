// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage renders correctly 1`] = `
<Layout>
  <SEO
    description=""
    keywords={
      Array [
        "gatsby",
        "application",
        "react",
      ]
    }
    lang="en"
    meta={Array []}
    title="Products"
  />
  <h1>
    Products
  </h1>
  <ProductList />
</Layout>
`;
